transport/cluster.go

```diff
@@ -0,0 +1,268 @@
+package transport
+
+import (
+	"fmt"
+	"log"
+	"net"
+	"strconv"
+	"sync/atomic"
+
+	"scylla-go-driver/frame"
+	. "scylla-go-driver/frame/request"
+	. "scylla-go-driver/frame/response"
+)
+
+var (
+	peerQuery = Statement{
+		Content:     "SELECT peer, data_center, rack, tokens FROM system.peers",
+		Consistency: frame.ONE,
+	}
+
+	localQuery = Statement{
+		Content:     "SELECT rpc_address, data_center, rack, tokens FROM system.local",
+		Consistency: frame.ONE,
+	}
+)
+
+func makePeerStatement(addr frame.Inet) Statement {
+	return Statement{
+		Content:     "SELECT peer, data_center, rack, tokens FROM system.peers WHERE peer = " + net.IP(addr.IP).String(),
+		Consistency: frame.ONE,
+	}
+}
+
+func InetToString(i frame.Inet) string {
+	return string(i.IP) + ":" + strconv.Itoa(int(i.Port))
+}
+
+const (
+	nodeAddr   = 0
+	nodeDC     = 1
+	nodeRack   = 2
+	nodeTokens = 3
```

Comment:
What's that?

Reply:
These represent specific indexes of columns returned from queries necessary for refresh topology. I will comment it so it becomes more readable.